api/src/util/project.ts

```diff
@@ -7,17 +7,17 @@ import { importFresh } from './import'
 
 // --- General wrappers around project config ---
 
-export function getStudioConfig() {
+export const getStudioConfig = () => {
```

Comment:
I'd prefer to keep all of these as proper functions instead of having them as arrow functions